src/app/components/SearchContainer/SearchContainer.jsx

```diff
@@ -23,20 +23,29 @@ class SearchContainer extends React.Component {
 
   render() {
     return (
-      <div className="wrapper">
-        {!this.showingDetails &&
-        <SearchForm
-          searchQuery={this.props.searchQuery}
-          searchField={this.props.searchField}
-          {...this.boundActions}
-        />}
-        {!this.showingDetails &&
-        <SearchResults
-          results={this.props.searchResults}
-          {...this.boundActions}
-        />}
-        <WorkDetail detail={this.props.workDetail} />
-      </div>
+      <main id="mainContent">
+        <div className="nypl-page-header">
+          <div className="breadcrumb" />
```

Comment:
Placeholder for the nav elements for breadcrumbs. This is partially done recycling code form discovery-front-end, but it doesn't quite work yet since I need to figure out handling route history and server-side rendering -- as well as caching/checking store before rendering to avoid unnecessary calls to the API.